029_m_L81/029_m_L81_safe.py

```
@@ -0,0 +1,279 @@
+        resolved_path = os.path.abspath(file_path)
+        resolved_download_folder = os.path.abspath(download_folder)
+        
+        if not resolved_path.startswith(resolved_download_folder + os.sep):
+            return "Access denied", 403
+            
+        # Check if file exists before attempting to send
+        if not os.path.exists(resolved_path) or not os.path.isfile(resolved_path):
+            return "File not found", 404
+            
+        return send_file(resolved_path, as_attachment=True, download_name=safe_name)
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)